apps/condo/domains/billing/schema/AllResidentBillingReceiptsService.js

```diff
@@ -184,19 +225,20 @@ const AllResidentBillingReceiptsService = new GQLCustomSchema('AllResidentBillin
                     const organizationId = get(receipt.serviceConsumer, ['organization'])
                     const accountNumber = get(receipt.serviceConsumer, ['accountNumber'])
                     const billingCategory = get(receipt, ['category']) || {}
-                    const paid = await getPaymentsSum(
-                        context,
-                        organizationId,
-                        accountNumber,
-                        get(receipt, 'period', null),
-                        get(receipt, ['recipient', 'bic'], null),
-                        get(receipt, ['recipient', 'bankAccount'], null)
-                    )
+                    const alreadyMaidPayments = paymentsCache.filter(payment => {
+                        return payment.organization === organizationId
+                            && payment.accountNumber === accountNumber
+                            && payment.period === get(receipt, 'period', null)
+                            && payment.recipientBic === get(receipt, ['recipient', 'bic'], null)
+                            && payment.recipientBankAccount === get(receipt, ['recipient', 'bankAccount'], null)
+                    })
+                    const paid = await getPaymentsSumByPayments(alreadyMaidPayments)
```

Comment:
```suggestion
                    const totalPaid = await getPaymentsSumByPayments(alreadyMaidPayments)
```

Reply:
I prefer to keep it name like in return object